apifortress/README.md

```diff
@@ -0,0 +1,54 @@
+## Overview
+
+The API Fortress integration is designed to bring your API Fortress test results into DataDog with ease. The process, once set up, is automated. Users can specify either failures only or all test results. Tests that are executed via the scheduler or via external API call will then feed their data into DataDog for visualization.
```

Comment:
A bit nit picky, but do we want to call out `visualization` here? Since with DD you can of course visualize/alert/etc on the data coming in.